guides/10_other-tutorials/building-mcp-server-with-gradio.md

```diff
@@ -209,3 +206,82 @@ To use this MCP Server with Claude Desktop (as MCP Client):
 Now, when you ask Claude about generating an image or transcribing audio, it can use your Gradio-powered tools to accomplish these tasks.
 
 
+## Troubleshooting your MCP Servers
+
+The MCP protocol is still in its infancy and you might see issues connecting to an MCP Server that you've built. We generally recommend using the [MCP Inspector Tool](https://github.com/modelcontextprotocol/inspector) to try connecting and debugging your MCP Server.
```

Comment:
Can we build a version of this and ship with gradio? e.g. `mcp/inspect` ?

Reply:
I think it's better to point users to the official debugger. I don't think we should be maintaining a rapidly-evolving inspector tool ourselves (and if there are issues with the inspector tool, then that could complicate debugging -- is the issue with our inspector tool or with the developer's code? What we could do instead is mention the `gradio_api/mcp/schema` route as an intermediate way to inspect your endpoint's schema